com/laboon/World.java

```diff
@@ -127,7 +127,22 @@ public String toString() {
 			}
 			toReturn += "\n";
 		}
-		return toReturn;
+		return toReturn;// this code is bad, and you should feel bad */
```

Comment:
I made deliberately inefficient, so I don't feel too bad.  I think I did a good job of making it poorly written!